basketball_reference_web_scraper/client.py

```diff
@@ -212,6 +212,35 @@ def team_box_scores(day, month, year, output_type=None, output_file_path=None, o
     )
     return output_service.output(data=values, options=options)
 
+def get_roster(team, year=None, output_type=None, output_file_path=None, output_write_option=None, json_options=None):
```

Comment:
Let's make `year` a non-optional argument. I don't think there's a pattern for any `year` values being optional (intentionally).